rainier-core/src/main/scala/com/stripe/rainier/core/RandomVariable.scala

```diff
@@ -127,6 +127,8 @@ class RandomVariable[+T](val value: T, val targets: Set[Target]) {
       def gradient(index: Int) = outputs(index + 1)
     }
 
+  lazy val densityValue: Real = targetGroup.base
```

Comment:
This will be incorrect if there are placeholders but it's fine for now...

Reply:
👍